### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by jmonterrubio**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+python: "3.7"
+os: linux
+services:
+  - docker
+install:
+  - pip install pipenv
+  - pipenv sync
+env:
+  jobs:
+
```
Comment (dortegau) [inline .travis.yml]: There are some empty lines :)

Comment (jmonterrubio) [reply .travis.yml]: done

**Review by ablancolopez — APPROVED** [review]

File: molecule/default/verify.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+# This is an example playbook to execute goss tests.
+# Tests need distributed to the appropriate ansible host/groups
+# prior to execution by `goss validate`.
+
+- name: Verify mapnik
+  hosts:
+    - mapnik
+  become: true
+  vars:
+    goss_version: v0.3.11
```
Comment (dortegau) [inline molecule/default/verify.yml]: There is a brand new goss v0.3.13 :p

**Review by jperera — APPROVED** [review]

File: molecule/default/verify.yml (hunk 2 above)
Comment (jmonterrubio) [reply molecule/default/verify.yml]: done

**Review by ablancolopez — APPROVED** [review]

**Review by sorobon — APPROVED** [review]

**Review by vicsufer — APPROVED** [review]

**Pull request merged**